docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -323,4 +324,26 @@ realm-name:realm-two
 
 There are other options that can be specified for converting a filesystem realm, the details of which can be found using the `./bin/elytron-tool.sh filesystem-realm-integrity -h` command. 
 
+== Updating WildFly Elytron Tool Generated CLI Scripts for Domain Mode
+As mentioned above, some of the WildFly Elytron Tool commands create a CLI script that can be used to add resources to
+the Elytron subsystem. Some examples of these CLI scripts include:
+
+. The <FILESYSTEM-REALM-NAME>.cli script generated when the `filesystem-realm-encrypt` function with the WildFly Elytron Tool is used to encrypt an existing filesystem realm.
+. The <FILESYSTEM-REALM-NAME>.cli script generated when the `filesystem-realm-inregrity` function with the WildFly Elytron Tool is used to add integrity checking to existing filesystem realms.
+. The <FILESYSTEM-REALM-NAME>.sh script generated when the `filesystem-realm` function with the WildFly Elytron Tool is used to create a filesystem realm from a legacy properties file.
+
+However, that CLI script can only be used directly if the server is running in Standalone mode. For Domain mode
```

Comment:
s/that CLI script/the generated CLI script

Reply:
fixed!